oauth2_http/pom.xml

```
@@ -23,57 +23,19 @@
 
   <profiles>
     <profile>
-      <!-- This profile is used to enable GraalVM native image testing -->
-      <id>native</id>
-      <dependencies>
-        <dependency>
-          <groupId>org.junit.vintage</groupId>
-          <artifactId>junit-vintage-engine</artifactId>
-          <version>5.10.0</version>
-          <scope>test</scope>
-        </dependency>
-        <dependency>
-          <groupId>org.graalvm.buildtools</groupId>
-          <artifactId>junit-platform-native</artifactId>
-          <version>0.9.28</version>
-          <scope>test</scope>
-        </dependency>
-      </dependencies>
+      <id>native-test</id>
       <build>
         <plugins>
           <plugin>
-            <groupId>org.apache.maven.plugins</groupId>
-            <artifactId>maven-surefire-plugin</artifactId>
-            <!-- Must use older version of surefire plugin for native-image testing. -->
-            <version>2.22.2</version>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-surefire-plugin</artifactId>
+          <version>3.0.0-M5</version>
```

Can we use the version defined in java-shared-config? Currently `3.2.1`: https://github.com/googleapis/java-shared-config/blob/main/pom.xml#L67
```suggestion
          <version>${surefire.version}</version>
```

---

Thanks! Done.